### DISCUSSION TITLE
ComboBox inside a ModalBody component will cut off dropdown

### DISCUSSION THREAD

**Discussion by hey-robin** [twilio-labs/paste#2622] [Q&A] [answered]
Hi all,

I am noticing some strange behavior in the ComboBox dropdown when it's used inside a ModalBody component. Long dropdown lists will be cut off. Is this an intentional design choice? It seems like the expected behavior should be similar to the Select component.

I've created a codesandbox with and example of both ComboBox and Select: https://codesandbox.io/s/silly-stitch-t0j1j6?file=/src/App.tsx

If this is a bug, I can submit it as such. 
Thanks!

**Comment by shleewhite** [comment]
Hi @robnit !

This is a known issue, we have a ticket in the backlog to fix it: https://issues.corp.twilio.com/browse/DSYS-2794. We have to refactor Combobox so that it uses a Portal to put the listbox on top of whatever container it is in.

If it is OK, I am going to close the issue you opened with a note pointing back to this post. When the bug is fixed, we will post an update here.

**Reply by hey-robin** [reply]
Glad to hear it's a known issue and has a corresponding ticket! I'm ok with you closing the issue and pointing to this post.

Is there a known ETA for this fix? I ask because my team is working on a feature that will expose this behavior. Unfortunately, it currently has a GA deadline of 9/30 https://issues.corp.twilio.com/browse/KYC-1119

**Reply by shleewhite** [reply]
Unfortunately no ETA yet. I have moved the ticket to the top of the backlog, so it could potentially be prioritized in the 10/18 release.

**Reply by hey-robin** [reply]
If we have the time and resources, is this a ticket that can be fixed by myself or another engineer on our team? I'm unsure if Paste is maintained exclusively by an internal team or if it accepts community contributions.

**Reply by shleewhite** [reply]
Definitely! We really appreciate contributions.

**Comment by shleewhite** [comment, marked as answer]
Hi @robnit!

We have recently published a fix for this issue. Please upgrade either `@twilio-paste/combobox` to `13.0.3` or `@twilio-paste/core` to `17.0.5`.

Let me know if you continue to have issues! Thank you again for letting us know about this bug.
